recipes/rtmidi/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,9 @@
+#include <RtMidi.h>
+
+int main (void) {
+    try {
+        RtMidiOut *midiout = new RtMidiOut();
```

Comment:
Don't forget to clean after you're done :smile: 
```suggestion
        RtMidiOut *midiout = new RtMidiOut();
        delete midiout;
```

Reply:
oups....